packages/calypso-build/webpack/minify.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * External Dependencies
+ */
+const TerserPlugin = require( 'terser-webpack-plugin' );
+
+module.exports = ( { shouldMinify, cache, parallel, sourceMap, terserOptions } ) => ( {
+	minimize: shouldMinify,
```

Comment:
This is just pass-through, so I'd opt to skip it, since it will make the export cleaner.

Reply:
(To clarify, this will allow us to either just export `new TerserPlugin( /*...*/ )` as the only symbol, or rename it from `minimizer` to `plugins` to be more consistent with other shards.)

Reply:
what about wrapping it in array in case the need arises for other minification plugins?

Reply:
Classic prompt for me to play the YAGNI card :grin: -- If that need arises, we can reconsider.

Bur for now (and potentially in the long run), I think it's okay to keep the information whether or not to minify confined to the consumer and not propagate it to the individual minfier :slightly_smiling_face:

Reply:
(I noticed only after writing the previous comment that you'd already made the change. Thanks, and disregard it then :grin: )